tests/foreman/ui/test_repository.py

```diff
@@ -641,3 +653,281 @@ def test_positive_update_custom_ostree_repo_url(self):
                     repo_name, 'url', FEDORA23_OSTREE_REPO
                 )
             )
+
+    @tier1
+    def test_positive_download_policy_displayed_for_yum_repos(self):
+        """Verify that YUM repositories can be created with download policy
+
+        @id: 8037a68b-66b8-4b42-a80b-fb08495f948d
+
+        @Assert: Dropdown for download policy is displayed for yum repo
+        """
+
+        with Session(self.browser) as session:
+            session.nav.go_to_select_org(self.session_org.name, force=False)
+            self.products.search_and_click(self.session_prod.name)
+            self.repository.navigate_to_entity()
+            self.repository.click(locators['repo.new'])
+            self.repository.text_field_update(
+                common_locators['name'], gen_string('alphanumeric'))
+            self.repository.select(locators['repo.type'], 'yum')
+
+            dropdown = self.repository.find_element(
+                locators['repo.download_policy'])
+
+            self.assertIsNotNone(dropdown)
+
+    @tier1
+    def test_positive_create_with_download_policy(self):
+        """Create YUM repositories with available download policies
+
+        @id: 8037a68b-66b8-4b42-a80b-fb08495f948d
+
+        @Assert: YUM repository with a download policy is created
+        """
+
+        with Session(self.browser) as session:
+            for policy in DOWNLOAD_POLICIES:
+                with self.subTest(policy):
+                    self.products.search_and_click(self.session_prod.name)
```

Comment:
@oshtaier I tried without that line, but got errors saying "repo" locator not found, I guess `make_repository` is not changing to proper page automatically because it has no `prod` argument.